internal/pkg/config/config.go

```diff
@@ -127,6 +128,34 @@ func newDefaultWalletConfig() *WalletConfig {
 	}
 }
 
+// DrandConfig holds all configuration options related to pulling randomness from Drand servers
+type DrandConfig struct {
+	// Addresses are are drand server addresses in the format
+	Addresses []string `json:"addresses"`
+	// Secure is whether or not the drand address are secure (e.g. TLS)
+	Secure bool `json:"secure"`
+	// DistKey is the distributed public key of the server group expressed as hex encoded coefficients
+	DistKey []string `json:"distKey"`
+}
+
+func newDefaultDrandConfig() *DrandConfig {
+	return &DrandConfig{
+		Addresses: []string{
+			"localhost:8080",
+			"localhost:8081",
+			"localhost:8082",
+			"localhost:8083",
+			"localhost:8084",
+		},
+		Secure: false,
+		DistKey: []string{
+			"8d98373363b8b62da5d5ef24accd81a35254a37c2c706989ff425c7a0ac24abee024721db6e03478831523fb7bf02cad",
```

Comment:
Where does this key come from?  A local test network like the addresses? The public network's real key?

Reply:
It's a local network. It's actually an old local network. I'll remove these for now.